core/kubernetes/setup.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+echo "Setup Kubernetes..."
+kubectl create configmap console-config --from-file=./config/openhim-console/default.json
+kubectl create configmap core-config --from-file=./config/openhim-core/development.json
+kubectl create configmap hapi-fhir-config --from-file=./config/hapi-fhir/hapi.properties
```

Comment:
These can be moved to a `configMapGenerator` in the kustomization file so we can be more declarative.